localstack/packages/core.py

```diff
@@ -112,7 +112,7 @@ def _install(self, target: InstallTarget) -> None:
         download_and_extract(
             download_url,
             retries=3,
-            tmp_archive=os.path.join(config.dirs.tmp, archive_name),
+            tmp_archive=os.path.join(dirs.tmp, archive_name),
```

Comment:
nit: i think we should keep the `config.dirs` approach, as this is the typical pattern in the codebase.